Tests/ReactiveSwiftTests/SignalProducerSpec.swift

```diff
@@ -1962,6 +1962,59 @@ class SignalProducerSpec: QuickSpec {
 				let result = producer.single()
 				expect(result?.value) == 1
 			}
+			
+			context("with interval") {
+				
+				it("should send values at the given interval until hitting the limitation") {
```

Comment:
This isn't a very accurate description of the test. It's testing the retrying, not testing the limitation.